examples/tutorials/data/hawc.py

```diff
@@ -100,6 +100,13 @@
 
 data_store.info()
 
+######################################################################
+# In this dataset, there are unfortunately overlapping GTIs. So
+# we are disabling the standard check of overlapping of GTIs.
+
+from gammapy.data import utils
+
+utils.CHECK_OVERLAPPING_TI = False
```

Comment:
What's the problem here ? is it specific to this HAWC dataset or something related to event types.
HAWC users should not have to set such global property. Opinion @LauraOlivera  ?

Reply:
This is an issue with our HAWC datasets, rather to HAWC in general

Reply:
why?